### PR TITLE
contents: bengali translation added for trivial js question

### PR DESCRIPTION
**Pull request by asifurrahaman754**
I have added the Bengali translation for js trivial question until map, foreach question. I will add all of them soon, as well as the HTML and CSS question

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -26,14 +26,14 @@ module.exports = {
         src: 'img/logo.svg',
       },
       items: [
-        {to: 'introduction', label: 'Start reading', position: 'left'},
-        {to: 'coding', label: 'Coding', position: 'left'},
+        { to: 'introduction', label: 'Start reading', position: 'left' },
```
Comment (yangshun) [inline website/docusaurus.config.js]: @asifurrahaman754 please don't make any unrelated changes. I recommend you install the Prettier extension for your IDE and let Prettier do the formatting based on the repo's Prettier config.

File: website/i18n/bn/javascript-questions.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+// doubled = [2, 4, 6]
+```
+
+`.forEach` এবং `.map()` এর মধ্যে প্রধান পার্থক্য হল `.map()` একটি নতুন অ্যারে প্রদান করে। আপনার যদি আসল অ্যারে পরিবর্তন করতে না চান, তাহলে `.map()` বেছে নিন। কিন্তু আপনি যদি কেবল একটি অ্যারের উপর পুনরাবৃত্তি করতে চান, তাহলে `forEach` বেছে নিন।
+
+###### রেফারেন্স
+
+- https://codeburst.io/javascript-map-vs-foreach-f38111822c0f
+
```
Comment (yangshun) [inline website/i18n/bn/javascript-questions.md:268]: Remove these empty lines. As mentioned, if you have the Prettier extension installed, it'd help you format all these.

**Review by yangshun — APPROVED** [review]

File: website/docusaurus.config.js (hunk 1 above)
Comment (asifurrahaman754) [reply website/docusaurus.config.js]: ok, got it
